### PR TITLE
Fix buildSrc/AbstractProjectGeneratorTask task properties

### PR DESCRIPTION
**Pull request by eskatos**
### Context
Follow up to #3574 and #3575

This PR reduces the number of warnings but leaves the`CacheableAsciidoctorTask` task untouched.

[CI Build](https://builds.gradle.org/viewLog.html?buildId=9523133) ✅

### CODE REVIEW

File: buildSrc/src/main/groovy/org/gradle/testing/performance/generator/tasks/AbstractProjectGeneratorTask.groovy [hunk 1]
```diff
@@ -30,27 +36,62 @@ import org.gradle.testing.performance.generator.TestProject
...
+    @Input
     List<String> additionalProjectFiles = []
-    final List<TestProject> projects = []
+
+    @Internal
+    String buildSrcTemplate
+    @Internal
     List<String> rootProjectTemplates = ['root-project']
+    @Internal
     List<String> subProjectTemplates = ['project-with-source']
```
Comment (lptr) [inline buildSrc/src/main/groovy/org/gradle/testing/performance/generator/tasks/AbstractProjectGeneratorTask.groovy]: How are these inputs tracked?

Comment (eskatos) [reply buildSrc/src/main/groovy/org/gradle/testing/performance/generator/tasks/AbstractProjectGeneratorTask.groovy]: The actual content of used templates is tracked here: https://github.com/gradle/gradle/pull/3606/files/ad2f09a2182ec43f13767d8c2bc69cfa30d59b40#diff-3df88e2eceb4da23ddeed01dbb250481R84

Comment (lptr) [reply buildSrc/src/main/groovy/org/gradle/testing/performance/generator/tasks/AbstractProjectGeneratorTask.groovy]: Roger. You can add this info as the `value()` of the `@Internal` annotation.

**Comment by lptr** [comment]
LGTM!

**Pull request merged**